src/Model.jl

```diff
@@ -16,11 +16,13 @@ struct Model{T <: Real}
     dim::Int # Dimension of the system; 3 unless `lattice` has zero columns
 
     # Electrons, occupation and smearing function
-    n_electrons::Int
+    n_electrons::Int # not necessarily consistent with `atoms` field
     spin_polarisation::Symbol  # :none, :collinear, :full, :spinless
     temperature::T
     smearing::Union{Nothing, Function} # see smearing_functions.jl for some choices
 
+    atoms # Vector of pairs AtomType => vector of vec3 (positions, fractional coordinates)
```

Comment:
Also: Personally I prefer `nothing` to disable stuff, because the compiler has an easier time optimising it away. Not that it matters in this case ...